libsignal-protocol/src/address.rs

```diff
@@ -52,6 +58,19 @@ impl<'a> Address<'a> {
     }
 
     pub fn device_id(&self) -> i32 { self.raw.device_id }
+
+    pub fn name(&self) -> &str {
```

Comment:
I know that there is another method called `bytes() -> &[u8]` but since i would use that for impl `Partial?{Ord,Eq}` traits, so i added it.

Reply:
What's wrong with just comparing the `&[u8]` when implementing `PartialEq` and friends?